packages/arb-token-bridge-ui/tests/e2e/specs/withdrawNativeToken.cy.ts

```diff
@@ -61,7 +61,7 @@ describe('Withdraw native token', () => {
           ETHToWithdraw = Number((Math.random() * 0.001).toFixed(5)) // generate a new withdrawal amount for each test-run attempt so that findAllByText doesn't stall coz of prev transactions
           cy.login({ networkType: 'childChain' })
           cy.typeAmount(ETHToWithdraw)
-          cy.findMoveFundsButton().click()
+          cy.startTransfer({ shouldConfirmInMetamask: false })
```

Comment:
i find this command name confusing
i only realised it was used for looking for the move funds button without clicking it after reading the command function itself

perhaps we can simply add some wait to the `findMoveFundsButton` function itself?